Packs/Syslog/Integrations/Syslogv2/Syslogv2.py

```diff
@@ -265,12 +255,14 @@ def prepare_globals_and_create_server(port: int, message_regex: Optional[str], c
         message_regex (Optional[str]): Regex. Will create incident only if Syslog message matches this regex.
         certificate (Optional[str]): Certificate. For SSL connection.
         private_key (Optional[str]): Private key. For SSL connection.
+        incident_type: Optional[str]: The incident Type.
 
     Returns:
         (StreamServer): Server to listen to Syslog messages.
     """
-    global MESSAGE_REGEX
+    global MESSAGE_REGEX, INCIDENT_TYPE
```

Comment:
no need for incident type to be global, please just use it as local var

Reply:
This is the original design of the integration (Tom implemented it like that and forgot to add)
Changing it will be more confusing and unnecessary work. I think we should keep it as is

Reply:
its a mistake, you dont reassign the global var anyway so no need to set it as such from the start

you can change that as well while you're here if you wish